shared/js/abp-preprocessed.es6.js

```diff
@@ -140,3 +143,40 @@ function getVersionParam () {
 
     return versionParam
 }
+
+function fetchSurrogateCode () {
+    let trackersSurrogateListEtag = settings.getSetting('trackersSurrogateList-etag') || ''
+	// reset etag to get a new list copy if we don't have data
+	if (!trackersSurrogateList || !trackersSurrogateListEtag) trackersSurrogateListEtag = ''
+	// load surrogates list
+	load.loadExtensionFile({url: constants.trackersSurrogateList, etag: trackersSurrogateListEtag, source: 'external'}, (listData, response) => {
```

Comment:
Style comment: generally this would look like:
```javascript
load.loadExtensionFile({
  url: constants.trackersSurrogateList,
  etag: trackersSurrogateListEtag,
  source: 'external'
}, (listData, response) => {
......
```

Reply:
works for me